scripts/ci.sh

```diff
@@ -19,6 +19,7 @@ rm -rf ${PWD}/build
 # Frameworks are only supported on iOS8 and later
 if [[ ! ${SIMULATOR} =~ .*OS=7.* ]]; then
   env NSUnbufferedIO=YES xcodebuild build -derivedDataPath=${PWD}/build/KIFFramework -scheme KIFFramework -destination "platform=iOS Simulator,${SIMULATOR}" | xcpretty -c
```

Comment:
I'd just get rid of the building command, because this is implied behavior from the test command that happens directly after this.